frontend/src/containers/CommitteeApplication/index.js

```diff
@@ -3,7 +3,7 @@ import Application from "./Application";
 
 class CommitteeApplication extends Component {
   componentDidMount() {
-    this.initializeValue();
+    setTimeout(this.initializeValue, 0);
```

Comment:
Curious, how does setTimeout with 0, differ from the old code?

Reply:
This is because of https://github.com/jaredpalmer/formik/issues/930.
Since you cannot call any of Formiks state-changing methods until after Formik has its `didMount` flag set to `true`.
Adding a `setTimeout` makes sure that Formiks lifecycle methods have been executed before we call `setFieldValue`.
See [here](https://stackoverflow.com/questions/33955650/what-is-settimeout-doing-when-set-to-0-milliseconds/33955673) on why this works.